t/blocker-Repositories.t

```diff
@@ -319,4 +319,76 @@ is $yum->_check_yum_repos() => { $unvetted => 1, $rpms_from_unvetted => 1, $inva
     no_messages_seen();
 }
 
+{
+    note 'Testing pacakges installed without associated repos blocker';
+
+    my $stdout = <<'EOS';
+Loaded plugins: fastestmirror, universal-hooks
+Loading mirror speeds from cached hostfile
+ * cpanel-addons-production-feed: 184.94.196.92
+ * epel: mirror.compevo.com
+Extra Packages
```

Comment:
Should probably have a line with wrap on all three given you accounted for all three wrapping?